camus-kafka-coders/src/main/java/com/linkedin/camus/etl/kafka/coders/JsonStringMessageDecoder.java

```diff
@@ -87,6 +88,15 @@ public void init(Properties props, String topicName) {
       else if (timestampFormat.equals("unix_milliseconds")) {
         timestamp = jsonObject.get(timestampField).getAsLong();
       }
+      // Else if timestampFormat is 'ISO-8601', parse that
+      else if (timestampFormat.equals("ISO-8601")) {
+        String timestampString = jsonObject.get(timestampField).getAsString();
+        try {
```

Comment:
I was just being consistent with what it does when normal parsing fails. If we want different behaviour, we should probalby change all of them as a seperate pull request.